components/server/src/migration/user-to-team-migration-service.ts

```diff
@@ -149,11 +152,20 @@ export class UserToTeamMigrationService {
         ]);
         log.info(ctx, "Migrated usage data.", { teamId: team.id, result });
 
-        result = await conn.query("UPDATE d_b_stripe_customer SET attributionid = ? WHERE attributionid = ?", [
-            newAttribution,
-            oldAttribution,
-        ]);
-        log.info(ctx, "Migrated stripe customer data.", { teamId: team.id, result });
+        const stripeCustomer = await conn.query(
```

Comment:
Should this happen before we migrate the Usage data? Or does it matter? I'm mostly thinking that if the migration gets interrupted, is it problematic if we've migrated the usage but not the user?

Reply:
I don't think it improves the situation in case a migration gets interrupted. Ideally, we would run the migration in a transaction, but our DB layer doesn't allow to pass down a connection.